src/plugins/auto/plugin.cpp

```diff
@@ -459,6 +459,7 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     if (configIter != fullConfig.end() && configIter->second == InferenceEngine::PluginConfigParams::CUMULATIVE_THROUGHPUT) {
         configIter->second = InferenceEngine::PluginConfigParams::THROUGHPUT;
         _LogTag = "AUTO";
+        _AutoSetToMulti = true;
```

Comment:
Why do we need to set `_AutoSetToMulti = true` in two places?

Reply:
Modified, only keep one `_AutoSetToMulti`, please review.